### PR TITLE
  filebeat/module/apache: Rename the field `apache2.module.error` to `apache.module.error`

### PR DESCRIPTION
**Pull request by herrBez**
## Proposed commit message

Rename field apache2.module.error to apache.module.error in the Dashboard

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Disruptive User Impact

None

## Author's Checklist

- [ ]

## Related issues

- #39480 

## Screenshots

image

The original saved search uses apache2.error.module, while the correct field apache.error.module (added by me) shows that it is actually populated with a meaningful value.

### CODE REVIEW

**Comment by shmsr** [comment]
Nice find. Can you also open a PR for the integrations repo? See: https://github.com/search?q=repo%3Aelastic%2Fintegrations+path%3A%2F%5Epackages%5C%2Fapache%5C%2Fkibana%5C%2F%2F+apache2&type=code

**Comment by shmsr** [comment]
Please add an entry to the changelog.

**Comment by herrBez** [comment]
Thank you. I think I addressed both points now :)

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -91,6 +91,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix cache processor expiries heap cleanup on partial file writes. {pull}38561[38561]
 - Fix cache processor expiries infinite growth when large a large TTL is used and recurring keys are cached. {pull}38561[38561]
 - Fix parsing of RFC 3164 process IDs in syslog processor. {issue}38947[38947] {pull}38982[38982]
+- apache: Rename field apache2.module.error into apache.module.error in the apache error visualization. {issue}39480[39480] {pull}39481[39481]
```
Comment (shmsr) [inline CHANGELOG.next.asciidoc:94]:
```suggestion
- Rename the field "apache2.module.error" to "apache.module.error" in Apache error visualization. {issue}39480[39480] {pull}39481[39481]
```

**Review by shmsr — CHANGES_REQUESTED** [review]

**Comment by shmsr** [comment]
Changes LGTM. However, I would like another member of my team to review this as well before we merge.

**Review by shmsr — APPROVED** [review]

**Comment by ishleenk17** [comment]
@shmsr @devamanv : I think this can be merged in beats as well?

**Comment by herrBez** [comment]
I see some failures in the build, but I hardly believe they are related with the change introduced here:
https://github.com/elastic/beats/pull/39481/checks?check_run_id=25345121364

**Comment by shmsr** [comment]
@herrBez CI is passing. Let's merge?

**Comment by herrBez** [comment]
Done. Thank you for the help and support!

**Pull request merged**